substrate/frame/revive/src/exec.rs

```diff
@@ -965,12 +972,13 @@ where
 	/// Run the current (top) frame.
 	///
 	/// This can be either a call or an instantiate.
-	fn run(&mut self, executable: E, input_data: Vec<u8>) -> ExecResult {
+	fn run(&mut self, executable: E, input_data: Vec<u8>) -> Result<(), ExecError> {
```

Comment:
Isn't that the same?

Reply:
`ExecResult` is an alias to `Result<ExecReturnValue, ExecError>` however we don't want to return `ExecReturnValue` anymore?